### PR TITLE
Removes extra d in delimiter 155

### LINKED ISSUE #155: Correct wrong format strings
```
contrib_regression=# select format('CREATE SEQUENCE %s START WITH %d', 'hola', 3+2);
ERROR:  unrecognized conversion type specifier "d"
contrib_regression=# select format('CREATE SEQUENCE %s START WITH %i', 'hola', 3+2);
ERROR:  unrecognized conversion type specifier "i"
contrib_regression=# select format('CREATE SEQUENCE %s START WITH %s', 'hola', 3+2);
```

This affects current code: https://github.com/CartoDB/cartodb-postgresql/blob/master/scripts-available/CDB_CartodbfyTable.sql#L168

### PR DESCRIPTION
**Pull request by gfiorav**
Fixes #155

To Do:

- [x] Fix bug
- [x] Write tests
- [x] CR (@rafatower)

### CODE REVIEW

File: scripts-available/CDB_CartodbfyTable.sql [hunk 1]
```diff
@@ -166,7 +166,7 @@ BEGIN
         AS seq INTO rec2;
 
       -- Reset sequence name
-      sql := Format('ALTER SEQUENCE %s RESTART WITH %d', rec2.seq::text, rec.max + 1);
+      sql := Format('ALTER SEQUENCE %s RESTART WITH %', rec2.seq::text, rec.max + 1);
```
Comment (rafatower) [inline scripts-available/CDB_CartodbfyTable.sql]: I don't think that code is correct
```
# select format('test %s %', 'foo', 42);
ERROR:  unterminated format specifier
```
Have you checked that the test actually fails without the fix? I don't think you're exercising that line with the test.

Have you checked there are no other %X wrong format specifiers?

**Comment by gfiorav** [comment]
@rafatower there was a problem where the `rec`'s `max` field wasn't set. It is now working.

File: test/CDB_CartodbfyTableTest_expect [hunk 2]
```diff
@@ -102,5 +102,12 @@ Table with non unique and null cartodb_id #148 cartodbfied fine
 3|
 4|2
 DROP TABLE
+CREATE TABLE
+
+DROP TABLE
+CREATE TABLE
+
+_cartodb_id0
+DROP TABLE
```
Comment (rafatower) [inline test/CDB_CartodbfyTableTest_expect]: I'd set the verbosity to NOTICE in the tests above to make sure the message "Could not initialize cartodb_id with existing values" is not raised during the tests.

**Comment by gfiorav** [comment]
@rafatower 

There is no `notice` level. http://www.postgresql.org/docs/9.4/static/app-psql.html

I'm setting it to `terse`.

**Comment by rafatower** [comment]
@gfiorav there are verbosity levels and logging levels http://www.postgresql.org/docs/9.0/static/runtime-config-logging.html

set the verbosity level:
```
\set VERBOSITY [terse|default|verbose]
```
set the logging level:
```
SET client_min_messages TO [debug|...];
```

**Comment by gfiorav** [comment]
Ah! I see.

Thanks for the info. I'll change that tomorrow

**Comment by gfiorav** [comment]
Ok, @rafatower, find the code refactored to your suggestions.

Much appreciated!

File: test/CDB_CartodbfyTableTest_expect [hunk 3]
```diff
@@ -102,5 +102,21 @@ Table with non unique and null cartodb_id #148 cartodbfied fine
...
+NOTICE:  Column cartodb_id already exists
+NOTICE:  Existing cartodb_id field does not have an associated sequence, renaming
+NOTICE:  Trying to recover data from _cartodb_id0 column
+
+DROP TABLE
+CREATE TABLE
+NOTICE:  Column cartodb_id already exists
+NOTICE:  Existing cartodb_id field is of invalid type text (need int2, int4 or int8), renaming
+NOTICE:  Trying to recover data from _cartodb_id0 column
+NOTICE:  Could not initialize cartodb_id with existing values: result of USING clause for column "cartodb_id" cannot be cast automatically to type integer (42804)
```
Comment (rafatower) [inline test/CDB_CartodbfyTableTest_expect]: this is exactly the message that should not be raised :((

Comment (rafatower) [inline test/CDB_CartodbfyTableTest_expect]: https://github.com/CartoDB/cartodb-postgresql/pull/175/files#diff-767fb5c7568f72bdf3da71fbe0d21ca4R180

Comment (gfiorav) [inline test/CDB_CartodbfyTableTest_expect]: Oh shoot! I apologise for the oversight. Kind of overloaded atm. I'll be back with it soon.

**Comment by gfiorav** [comment]
@rafatower It seems that the original code was missing an explicit `::integer` cast for the `ALTER TABLE %s ALTER cartodb_id TYPE int USING %I::integer` expression. Now it should be OK.

**Comment by rafatower** [comment]
:+1:

**Pull request merged**